client/src/components/modals/incorporate-login.vue

```diff
@@ -1,26 +1,32 @@
 <template>
-  <v-dialog v-model="showModal" max-width="60%" >
+  <v-dialog v-model="showModal" max-width="850px" persistent>
```

Comment:
Would it be easy to "attach" this v-dialog to its parent? If no, nvm.

Reply:
This is refactored a bit, i made a new common component that is a dialog itself, so this will change a bit.